io/plugins/eu.esdihumboldt.hale.io.jdbc.spatialite/src/eu/esdihumboldt/hale/io/jdbc/spatialite/reader/internal/SpatiaLiteInstanceReader.java

```diff
@@ -77,4 +81,14 @@ public void setSource(LocatableInputSupplier<? extends InputStream> source) {
 		internalProvider.setSource(new SpatiaLiteJdbcIOSupplier(new File(source.getLocation())));
 	}
 
+	/**
+	 * @see eu.esdihumboldt.hale.common.core.io.IOProvider#execute(eu.esdihumboldt.hale.common.core.io.ProgressIndicator,
+	 *      java.lang.String)
+	 */
+	@Override
+	public IOReport execute(ProgressIndicator progress, String resourceIdentifier)
+			throws IOProviderConfigurationException, IOException {
+		return null;
```

Comment:
I would guess for importing multiple SpatiaLite files this would have to be implemented?

If we do not support it I would be for throwing an `UnsupportedOperationException` and adding a comment instead of just returning `null`.

Reply:
Code modified to throw `UnsupportedOperationException`.